pkg/volume/persistent_claim/persistent_claim.go

```diff
@@ -80,6 +81,19 @@ func (plugin *persistentClaimPlugin) NewBuilder(spec *volume.Spec, pod *api.Pod,
 		return nil, err
 	}
 
+	// If a GID annotation is provided set the GID attribute.
+	if volumeGid, ok := pv.Annotations["GID"]; ok {
```

Comment:
> kubernetes.io/supplemental-gid or something similar.

Do you guys have any suggestions other than supplemental-gid ? supplemental is more for processes and users which can be members of more than one group but volumes (files) are only owned by one gid